equinix/helper_test.go

```diff
@@ -0,0 +1,179 @@
+package equinix_test
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/equinix/ecx-go/v2"
+)
+
+//‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+// Legacy test helper functions
```

Comment:
I hate this file, but it's frankly intended to be ugly and short-lived; this exists because of how our legacy code is structured.  A lot of what's in here will go away as we remove support for deprecated APIs, and the rest has clear alternatives that we can use rather than moving those files to separate packages and making them public.

As much as I dislike this file, having it here is essential for keeping this diff small and getting us in good shape for future smaller refactors.

Reply:
Thanks for the context 👍